ckan/config/environment.py

```diff
@@ -367,6 +367,9 @@ def template_loaded(template):
         template_paths, auto_reload=True, callback=template_loaded)
 
     # Create Jinja2 environment
+    cache_dir = config.get('jinja2_cache_dir', '/tmp/jinja_cache')
```

Comment:
See the `tempfile` module (https://docs.python.org/2/library/tempfile.html) to make a secure temporary directory as the default.